vision_agent/agent/vision_agent.py

```diff
@@ -136,10 +149,8 @@ def __init__(
             code_sandbox_runtime (Optional[str]): The code sandbox runtime to use.
         """
 
-        self.agent = (
-            OpenAILMM(temperature=0.0, json_mode=True) if agent is None else agent
-        )
-        self.max_iterations = 100
+        self.agent = AnthropicLMM(temperature=0.0) if agent is None else agent
+        self.max_iterations = 12
```

Comment:
NIT why 12 ? I looked at the code its for trying n number iterations to respond to a user chat, correct ? When does it do that ? like endpoint down or stuck in some loop ?

Reply:
12 is arbitrary, but occasionally it will get stuck in loops, for example it might try to run a program, get an error and then not fix the error correctly, re-run, re-fix, etc. It rarely needs to actually run for >10 iterations to accomplish a task